cmd/wallet-web/src/pages/OIDCShare.vue

```diff
@@ -327,6 +325,31 @@ export default {
       );
       this.vpToken = encodeURI(JSON.stringify(presentation));
     },
+    async verifyRPRegistration() {
+      const registrationUri = this.$route.query.registration_uri;
+      const registration = this.$route.query.registration;
```

Comment:
Since `$route.query` is accessible in the url, shouldn't we sanitize the values that we read from the it? This applies to many other places in our code...
cc @sudeshrshetty @rolsonquadras